### ISSUE TITLE
Art and Graphics

### ISSUE THREAD

**Issue by Ar13mis** [boomboxInc/Salt#33]
Discussion thread for the graphics and art of the game as well as a good home for art proposals.

**Comment by Ar13mis** [comment]
I took the liberty of drawing up a couple starting ships in the style I have a bit of skill working in. Here are the samples.

![testship](https://cloud.githubusercontent.com/assets/8703799/15455620/aec359a2-2016-11e6-9f1d-a6e633fa4137.png)

![testshipred](https://cloud.githubusercontent.com/assets/8703799/15455633/0b2600aa-2017-11e6-866f-cad7e4996ad2.png)

**Comment by Ar13mis** [comment]
As well as a theoretical obstruction.

![testmidplanet1](https://cloud.githubusercontent.com/assets/8703799/15455690/5d55930c-2019-11e6-85a5-05902dc6e6a6.png)

**Comment by ConnorGBrewster** [comment]
I like the ships! the obstruction looks good, although we will need ones that can grow pretty large

**Comment by Ar13mis** [comment]
Of course, but as the style does it work? I can make them pretty much as large as we would like, I just wanted to make sure the idea would work with everyone involved before I started making large numbers of them for concepting.

**Comment by ConnorGBrewster** [comment]
I think the style is fine, you're the artist so you get to define the style!

**Comment by Ar13mis** [comment]
New ship textures.

![testshipblue](https://cloud.githubusercontent.com/assets/8703799/15486127/aa4981cc-2100-11e6-8e1b-0e40a0d24a55.png)
![testshipred](https://cloud.githubusercontent.com/assets/8703799/15486128/aa597852-2100-11e6-8e85-ac565f58e8cb.png)
